rpc/src/v1/traits/parity.rs

```diff
@@ -241,5 +241,11 @@ build_rpc_trait! {
 		/// as well as checks the signature for chain replay protection
 		#[rpc(name = "parity_verifySignature")]
 		fn verify_signature(&self, bool, Bytes, H256, H256, U64) -> Result<RecoveredAccount>;
+
+		/// Returns logs matching given filter object.
+		/// Skip filling transaction hash for faster query.
+		#[rpc(name = "parity_getLogsLight")]
```

Comment:
I have no strong opinions on the naming but `getLogsLight` is not self-explanatory as long the documentation explains this I'm satisfied.

Otherwise, I propose `parity_getLogsNoTransactionHash`